content/docs/command-reference/exp/index.md

```diff
@@ -0,0 +1,44 @@
+# exp[eriments]
+
+A set of commands to generate and compare local _experiments_:
+[show](/doc/command-reference/exp/show),
+[checkout](/doc/command-reference/exp/show),
+[diff](/doc/command-reference/exp/diff), [run](/doc/command-reference/exp/run),
+[resume](/doc/command-reference/exp/resume), and
+[gc](/doc/command-reference/exp/gc).
+
+## Synopsis
+
+```usage
+usage: dvc exp[eriments] [-h] [-q | -v] {show,checkout,diff,run,resume,res,gc} ...
+
+positional arguments:
+  COMMAND
+    show                Print experiments.
+    checkout            Checkout experiments.
+    diff                Show changes between experiments in the DVC repository.
+    run                 Reproduce complete or partial experiment pipelines.
+    resume (res)        Resume checkpoint experiments.
+    gc                  Garbage collect unneeded experiments.
+```
+
+## Types of experiments
+
+DVC supports two types of experiments:
+
+1. Standalone experiments
+2. Checkpoints
+
+## Description
+
```

Comment:
This one will be pretty important, defining what DVC Experiments are exactly, and how DVC implements in general (more details in each subcommand).

Reply:
So actually we can leave the deep discussion on experiment management for a future guide/use case. Let's keep a very short  cmd ref index for now — we may even get rid of all these indices soon (discussed somewhere in #1944 I think).